lineman/app/components/group_page/group_page.scss

```diff
@@ -59,12 +59,23 @@
   @include card;
 }
 
+.group-page__threads {
+  padding: 10px 0 17px;
+  h2 {
```

Comment:
Rob that is not going to work. The lmo-card-heading does not have a margin-left for a reason. It relies on the padding of the container. If you look at the diff you will see I have removed a whole lot of negative margins, which I consider to be 'incorrect' css, 'incorrect' not because the script is ugly , but because it behaves differently in different browsers. It's un predictable. As the margins in the container need to be 0 to allow the separating lines between the threads to span 100%, I need to pad the headers for this case.  I personally don't think nesting is dumb it can be very powerful.  If you look at the BEM docs it emphasises that it is not absolute and does not work in all cases.  To me it is more important how it looks to the user and how it scales across browsers and different devices.  The css layer should be flexible and changing as the product changes.  Nesting a header is not going to stop that.  Feel free to refactor once it is in place.